lambda-io-app/jvm/src/main/scala/com/dwolla/lambda/IOLambda.scala

```diff
@@ -99,22 +122,30 @@ abstract class IOLambda[A, B](printer: Printer = Defaults.printer,
 object IOLambda {
   object Defaults {
     val printer: Printer = Printer.noSpaces
-    val executionContext: ExecutionContext = ExecutionContext.global
+    val executionContext: ExecutionContext = ExecutionContext.global // TODO change this to the IOApp default
```

Comment:
This TODO shouldn't block this PR; it's something that should be completed before publishing a non-milestone release.